tools/ci/run_tc.py

```diff
@@ -257,13 +257,33 @@ def setup_environment(args):
         assert args.channel is not None
         install_chrome(args.channel)
 
+    if "chromium" in args.browser:
+        install_chromium()
+
     if args.xvfb:
         start_xvfb()
 
     if args.oom_killer:
         start_userspace_oom_killer()
 
 
+def install_chromium():
```

Comment:
How about adding way to install the pinned revision using `./wpt install` and using that here?

If the pinned Chromium is only used (usable) by CI, then we should put `latest_chromium_revision.txt` somewhere under `tools/ci/`.

Reply:
I implemented it this way because we've always downloaded Chrome for testing in a way that is separate from `./wpt install`. If it is best for them to use the same install method, I can make that change.

Reply:
I think that's because Chrome can't be downloaded and installed easily on all platforms, and `./wpt install` should presumably work for all users. In CI we only need to support Linux.

Is the code path for installing Firefox also different? If not, the. Following that pattern should work. But if even Firefox isn't a single code path, then I'd not bother for Chromium either.